README.md

```diff
@@ -1,2 +1,30 @@
-# workshop
-This is a workshop to teach you modules packaging.
+# Modularity workshop
+
+## High-level agenda
+
+1. 1-minute-intro to modularity
+
+2. `modulemd.yaml`
+ * Guidelines
+ * Best practices (do we have this? is this guidelines?)
+ * Buildtime dependencies
+   * List of well-known modules (What is the status of platform module? I don't want to teach base-runtime)
```

Comment:
Platform is an ever evolving module/concept. The status depends on when you intend to deliver this workshop. At the moment there are no Platform builds.